LXStudio-IDE/Projects/te.lxp

```diff
@@ -427,22 +427,22 @@
         "centerPoint": false,
         "camera": {
           "active": false,
-          "radius": 13.5,
-          "theta": 0.0,
-          "phi": 0.0,
-          "x": 4.5,
-          "y": 0.0,
-          "z": 0.0
+          "radius": 1.6700490355594486E7,
+          "theta": -1.6295745824463665,
+          "phi": 0.2857836023904383,
+          "x": 776130.9353027344,
+          "y": 2626079.4140625,
+          "z": 54030.869079589844
```

Comment:
I believe these lines help the te.lxp project have our entire model in view by default (so new people do not need to scale out to many-micron view)

Reply:
📷👍🏼